src/website/app/demos/Link/examples/variants.js

```diff
@@ -1,7 +1,14 @@
 /* @flow */
 import React from 'react';
+import { createStyledComponent } from '../../../../../library/styles';
 import _Link from '../../../../../library/Link';
-import { DemoLayout } from './states';
+import _DemoLayout from '../components/DemoLayout';
+
+const DemoLayout = createStyledComponent(_DemoLayout, {
```

Comment:
I think you could use the [shared DemoLayout](https://github.com/mineral-ui/mineral-ui/blob/master/src/website/app/demos/shared/DemoLayout.js) Victoria made instead.

Reply:
Yep.